doc/rst/language/spec/statements.rst

```diff
@@ -671,13 +671,16 @@ A ``break`` statement cannot be used to exit a parallel loop
 
 ..
 
-   *Future*.
+.. note::
 
-   We expect to support a *eureka* concept which would enable one or
-   more tasks to stop the execution of all current and future iterations
-   of the loop.
+  *Future:*
+    
+    We expect to support a *eureka* concept which would enable one or
+    more tasks to stop the execution of all current and future iterations
+    of the loop.
 
-   *Example*.
+
+*Example*.
```

Comment:
This was the place I had issue, maybe would require help @ben-albrecht, If I remove the white space the `*Example` block is considered inside the note block because of indendation. 
Is there a way to mark finish of `..note::`block, Saw something like marking it with a whitespace and `..` but didn't give expected results.  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#inline-markup-recognition-rules

Reply:
I would suggest indenting the note block, e.g.

```rst
A ``break`` statement cannot be used to exit a parallel loop
:ref:`Forall`.

   *Rationale*.

   Breaks are not permitted in parallel loops because the execution
   order of the iterations of parallel loops is not defined.


   .. note::
  
    *Future:*
      
      We expect to support a *eureka* concept which would enable one or
      more tasks to stop the execution of all current and future iterations
      of the loop.


   *Example*.

   In the following code, the index of the first element in each row of
   ``A`` that is equal to ``findVal`` is printed. Once a match is found,
   the continue statement is executed causing the outer loop to move to
   the next row. 
```